package.json

```diff
@@ -20,7 +20,7 @@
     "modules"
   ],
   "dependencies": {
-    "detective": "3.1.0"
+    "detective": "git://github.com/shenanigans/node-detective.git"
```

Comment:
will not merge. This makes a floating dependency that is unsafe for any sort of use.